### PR TITLE
Raw Hex of Objects in Details Tab #616

### LINKED ISSUE #616: Raw Hex of Objects in Details Tab
It is often useful to get full hex of a transaction to copy and paste into a wallet or other transaction submission APIs. It would be nice if it were exposed under Details in the transaction view. While you're at it, block header raw hex would be cool too.

### PR DESCRIPTION
**Pull request by priyanshiiit**
Fix #616 

**Tasks**
---
- [x] Add raw block API
- [x] Add raw tx API
- [x] Add raw block in block details
- [x] Add raw tx in tx details

**Preview Images**
---
Updated frontend
Screenshot from 2021-08-02 20-29-32
---
API response
Screenshot from 2021-08-02 20-29-46
---
Updated frontend
Screenshot from 2021-08-02 20-30-18
---

**API response**
Screenshot from 2021-08-02 20-30-28

### CODE REVIEW

**Comment by wiz** [comment]
did you test this on firefox? looks like some tests failed

**Comment by TheBlueMatt** [comment]
Concept ACK. I guess if the hex is short (it always is for headers) it would be nice to include it in the GUI, but it doesn't matter *too* much. The block header detail view should include a link to the previous block with its hex, though, as far as I can tell there is no link to the previous block anywhere from the block details view.

**Comment by priyanshiiit** [comment]
> did you test this on firefox? looks like some tests failed

@wiz Yes, it works fine for me on firefox.

**Comment by wiz** [comment]
Hmm okay, seems it's a problem with the tests themselves, I will work with @knorrium on this

**Comment by wiz** [comment]
for starters can you please not commit the `package-lock.json` files - looks like you're using an old version of nodejs

**Comment by wiz** [comment]
okay let me clarify about the `package-lock.json` files - they are in git and we want them to be in git, but the problem is you are using an old version of nodejs (recommend you upgrade to the latest v16.5.0 using nvm-js) which is clobbering the package-lock.json files. and now you've deleted them entirely from git which isn't what we want since now the project won't build heh. so what you need to do to fix your PR is amend the original commit(s) you did that clobbered the `package-json.lock` files and then force-push your PR branch so that the clobbering never happened on that branch....

also in the future when you do pull requests, it's best to make a branch for each task, call the branch like `add-raw-hex-objects-in-details-tab` or whatever, and then do the PR from that task branch, it's not a good idea to do PRs from master

**Review by wiz — APPROVED** [review]
utACK for git workflow - we fixed the `package-lock.json` related issues, so it's off to @softsimon for his review next

File: frontend/angular.json [hunk 1]
```diff
@@ -1,5 +1,8 @@
 {
   "$schema": "./node_modules/@angular/cli/lib/config/schema.json",
+  "cli": {
+    "analytics": false
+  },
```
Comment (wiz) [inline frontend/angular.json:5]: This should not be committed

**Pull request merged**